components/speedreader/rust/lib/src/readability/src/nlp.rs

```diff
@@ -0,0 +1,121 @@
+use regex::{Regex, RegexSet};
+use std::cmp::max;
+use std::collections::HashSet;
+
+// A list of common abbreviations
+static EN_ABBREVIATIONS_LIST: [&str; 80] = [
```

Comment:
It'd be good to reference the source or methodology used to generate this list, so its completeness can be evaluated in the future

Reply:
Fixed in https://github.com/brave/brave-core/pull/10027/commits/29af44a144153288982d7bb859609cdad197aeb7